Packs/AzureDataExplorer/Integrations/AzureDataExplorer/AzureDataExplorer_description.md

```diff
@@ -28,15 +45,25 @@ to link Azure Data Explorer with Cortex XSOAR.
 
 At end of the process you'll see a message that you've logged in successfully.
 
-### Cortex XSOAR Azure App
```

Comment:
please keep it under the `Device Code Flow` section:)
but maybe as a section under it with ####